packages/eas-build-job/src/common.ts

```diff
@@ -16,6 +16,14 @@ export enum ArchiveSourceType {
   PATH = 'PATH',
 }
 
+export enum BuildResourceClass {
```

Comment:
This way allows to assignment ios resource class to android build, it should be separated

Reply:
fixed